localization/english/event_localization/wc_historical_artifacts_l_english.yml

```diff
@@ -0,0 +1,8 @@
+﻿l_english:
+
+  wc_historical_artifacts.0201.title:0 "A Warlock's Weapon"
+  wc_historical_artifacts.0201.desc:0 "At last, after a fierce battle, the corpse of the orc warlock lies on the ground admist our fallen. His weapon, a dark crystal, rests in his palm, emitting a sinister glow.\n\nAs I approach, a shiver runs down my spine. My gaze is locked on the artifact, drawn in by its hypnotic glow. It whispers at the edge of my mind, calling out to me. I bend down and reach for it. The instant my fingers brush the crystal, a searing pain shoots through me.\n\n"Aargh! What sorcery is this?!" I shout as I drop the crystal."
```

Comment:
You can refer the artifact in the localization with:

`[dark_crystal.GetName]` instead of just "dark crystal"